source/monitoring.txt

```diff
@@ -614,17 +604,16 @@ Source Connector JMX Metrics
 
        .. _kafka-monitoring-averge-record-size-span:      
 
-       .. tip:: Average Record Size Over a Span of Time
-
-          To calculate the average size of the records your sink connector
-          processed over a span of time, perform the following actions:
+       To calculate the average size of the records your sink connector
+       processed over a span of time, perform the following actions:
 
-          #. Determine the change in the value of the ``mongodb-bytes-read``
-             attribute for a span of time.
-          #. Determine the change in the value of the ``records`` attribute
-             for the same span of time you used for the preceding step.
-          #. Divide the change in the value of the ``mongodb-bytes-read`` attribute
-             by the change in the value of the ``records`` attribute.
+       #. Determine the change in the value of the
```

Comment:
This is rendering fine as a numbered list so no changes needed, but curiosity question: For inline markup, should we be writing ordered numbered lists as: 

> 1. 
>   #. 
>   #. 

Or will #. always render as numbers if it is not preceded by any other numerator?